core/Tracker/Db.php

```diff
@@ -30,6 +30,9 @@ abstract class Db
 
     protected $connection = null;
 
+    // this is used for indicate TransactionLevel Cache
+    public $supportsUncommitted = null;
```

Comment:
Is the TransactionLevel only used with Tracker\Db instances. If not we might also add that new variable to `Piwik\Db` maybe?

Reply:
I would say so it be good to do it for both as the regular DB uses it as well eg when archiving

Reply:
the test below also uses the regular DB

Reply:
This one still needs to be done. I don't think this is needed in here but rather in `Tracker/Mysqli` and `Tracker/pdo/mysql` DB?

Reply:
@tsteur aren't `Tracker/Mysqli` and `Tracker/PDO/Mysql` inheriting from `Tracker/Db`?

Reply:
They do indeed 👍